compiler/one-cmds/tests/onecc_neg_031.workflow.json

```
@@ -0,0 +1,29 @@
+{
+    "workflows": [
+        "codegen_wf"
+    ],
+    "codegen_wf": {
+        "steps": [
+            "import_tf",
+            "codegen"
+        ],
+        "import_tf": {
+            "one-cmd": "one-import-tf",
+            "commands": {
+                "input_path": "inception_v3.pb",
+                "output_path": "inception_v3.onecc_031.circle",
```

To avoid conflict with onecc_031 test.

```suggestion
                "output_path": "inception_v3.onecc_neg_031.circle",
```